benchmark/SqlBindingBenchmarks.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Configs;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.Benchmark
+{
+    public class SqlBindingBenchmarks
+    {
+        public static void Main()
+        {
+            BenchmarkRunner.Run<SqlInputBindingBenchmarks>(
+                ManualConfig
+                    .Create(DefaultConfig.Instance)
+                    .WithOptions(ConfigOptions.DisableOptimizationsValidator));
```

Comment:
Add comment about why you're doing this

Reply:
This config is actually not needed - will remove.